src/mappings/UController.ts

```diff
@@ -0,0 +1,19 @@
+import 'allocator/arena'
+export { allocate_memory }
+
+import { Entity, Value, store, crypto, ByteArray } from '@graphprotocol/graph-ts'
+
+import { MintReputation, BurnReputation, MintTokens } from '../types/UController/UController'
+
+export function handleMintReputation(event: MintReputation): void {
+    // Tracking reputation minted per account in native rep tracker
+
+    let dao = new Entity()
+    dao.setAddress('controllerAddress', event.address)
```

Comment:
why new dao here ?

Reply:
we should add member to already existing dao here

Reply:
Since dao entity has avatar as the unique identifier, it will update only the controllerAddress of an existing dao and not create a new dao if it already exists.
Computation to get entity from store and then updating controllerAddress if not already set is more than just setting the controllerAddress.

Reply:
> we should add member to already existing dao here

Reputation minted to founder member only emits mint event from the reputation contract. So adding members here was missing the founder members, hence I add the members as part of mint event in reputation contract

Reply:
> Since dao entity has avatar as the unique identifier, it will update only the controllerAddress of an existing dao and not create a new dao if it already exists.
> Computation to get entity from store and then updating controllerAddress if not already set is more than just setting the controllerAddress.

why to update controller address on the mint reputation event ?

Reply:
> > we should add member to already existing dao here
> 
> Reputation minted to founder member only emits mint event from the reputation contract. So adding members here was missing the founder members, hence I add the members as part of mint event in reputation contract

Though the Mint /Burn events of the Reputation contracts does not give you the information of which DAO this rep is belonging. You should do that via the U/Controller events